main/main.c

```diff
@@ -611,6 +611,7 @@ PHP_INI_BEGIN()
 #ifdef PHP_WIN32
 	STD_PHP_INI_BOOLEAN("windows.show_crt_warning",		"0",		PHP_INI_ALL,		OnUpdateBool,			windows_show_crt_warning,			php_core_globals,	core_globals)
 #endif
+	STD_PHP_INI_ENTRY("syslog.ascii_only",			"0",		 PHP_INI_SYSTEM,	OnUpdateBool,		syslog_ascii_only,		php_core_globals, 		core_globals)
```

Comment:
I would prefer to see something more generic like `syslog.encoding` to allow us also properly support UTF-8 - add BOM if selected. We also have functions valildation for UTF-8 and some other encoding in html.c in standard ext so it might be also good idea to use. This is just an initial thought so needs probably more thinking and it's open for discussion ofc. :)